students/12th/eunsulim/js/login.js

```diff
@@ -0,0 +1,16 @@
+// ID, PW 입력 시 로그인 버튼 활성화 기능
+const id = document.getElementsByClassName("inputId")[0];
+const pw = document.getElementsByClassName("inputPw")[0];
+const btn = document.getElementsByClassName('loginBtn')[0];
+
+const activeBtn = () => {
+    const length = id.value.length > 0 && pw.value.length > 0
```

Comment:
length 라는 변수 명이 적절하지 않은 것 같습니다!
변수 명만으로도 어떤 변수인지 알 수 있게 바꿔주세요!
그리고 input의 value는 string이라서 굳이 length > 0 이렇게 주시기 보다는
```suggestion
    const length = id.value && pw.value
```
이렇게 줄이실 수 있습니다!

Reply:
네 변수 수정했고, 코드 줄여서 다시 적용했습니다. 감사합니다.